src/containers/AccountConnection.jsx

```diff
@@ -13,13 +13,14 @@ class AccountConnection extends Component {
   constructor (props, context) {
     super(props, context)
     this.store = this.context.store
+    this.state = {
+      account: this.props.existingAccount
```

Comment:
I understand you needs here indeed. And what about using a flag like `newCreation=true`? When you create a very new account, your put a flag on it and then, when you get the created account from the stack, this flag will disappear automatically because not saved in the database. What do you think?